lib/pbench/cli/agent/commands/results/__init__.py

```diff
@@ -2,13 +2,21 @@
 
 import os
 import click
-import requests
+
+# import requests
 import shutil
-import sys
+
+# import sys
```

Comment:
Sorry!  Didn't notice that this PR is still in "draft" state.

Reply:
Sure.. I just commented that out because they are required in the code. But at the place where these are required, I have commented that out. I commented that code out coz that was not allowing me to pass pytests.